src/locales/pt-br/translation.json

```diff
@@ -0,0 +1,181 @@
+  "Search for Escalation Policy or User": "Pesquisar Política de Escalação ou Usuário",
+  "Select an open incident to merge into": "Seleccione um incidente aberto com o qual será combinado",
+  "Select dotdotdot": "Selecione...",
+  "Selected": "Selecionado",
+  "Service": "Serviço",
+  "Settings": "Configurações",
+  "Severity": "Severidade",
+  "Sign In": "Entrar",
+  "Signing into PagerDuty Live": "Entrando em PagerDuty Live",
+  "Since": "Desde",
+  "Snooze": "Suspender",
+  "Snooze for the following hours": "Suspender durante as próximas horas",
+  "Snooze Incident #X": "Suspender incidente #{{unresolvedIncidents.0.incident_number}}",
+  "Snooze until tomorrow at": "Suspender até amanhã às",
+  "Snooze X incidents": "Adiar {{unresolvedIncidents.length}} incidentes",
+  "snoozed": "adiado",
+  "Source": "Origem",
+  "State": "Estado",
+  "Status": "Estado",
+  "Summary": "Resumo",
+  "Sync with": "Sincronizar com",
+  "Synced with": "Sincronizado com",
+  "Team": "Equipe",
+  "Teams": "Equipes",
+  "The alerts of the selected incidents will be merged into a single incident": "Os alertas dos incidentes selecionados serão combinados em um único incidente",
+  "The remaining selected incidents will be resolved after the merge is complete": "Os restantes incidentes selecionados serão resolvidos, uma vez completa a combinação",
+  "Title": "Título",
+  "Triggered": "Acionado",
+  "triggered for": "acionado para",
+  "Unable to fetch account abilities": "Não foi possível obter as capacidades da conta",
+  "Unable to fetch current user details": "Não foi possível obter os dados atuais do usuário",
+  "Unable to fetch escalation policies": "Não foi possível obter as políticas de escalação",
+  "Unable to fetch extensions": "Não foi possível obter as extensões",
+  "Unable to fetch incidents": "Não foi possível obter os incidentes",
+  "Unable to fetch log entries": "Não foi possível obter as entradas de registro",
+  "Unable to fetch priorities": "Não foi possível obter as prioridades",
+  "Unable to fetch response plays": "Não foi possível obter os conjuntos de respostas",
+  "Unable to fetch services": "Não foi possível obter os serviços",
+  "Unable to fetch teams": "Não foi possível obter as equipes",
+  "Unable to fetch users": "Não foi possível obter os usuários",
+  "Unauthorized Access": "Acesso não autorizado",
+  "Unknown error while using PD API": "Erro desconhecido ao usar a API da PD",
+  "Update Incident Table": "Atualizar Tabela de incidentes",
+  "Update Priority": "Atualizar Prioridade",
+  "Update User Profile": "Atualizar Perfil de Usuário",
+  "Updated incident table settings": "Configurações de tabela de incidentes atualizadas",
+  "Updated user profile settings": "Configurações de perfil de usuário atualizadas",
+  "Urgency": "Urgência",
+  "User is not permitted to access this instance of PagerDuty Live": "O usuário não tem permissão para aceder a esta instância do PagerDuty Live",
```

Comment:
As a brazilian, I'm not used to aceder, instead I think we can use "acessar"

Reply:
```suggestion
  "User is not permitted to access this instance of PagerDuty Live": "O usuário não tem permissão para acessar a esta instância do PagerDuty Live",
```